common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -232,6 +232,22 @@ def get_html(self):
 
         track_url, transcript_language, sorted_languages = self.get_transcripts_for_student()
 
+        course = None
+        course_id = None
+
+        licensable = False
+        video_license = None
+        if settings.FEATURES.get('CREATIVE_COMMONS_LICENSING', False):
+            video_license = self.license
+            if hasattr(self.descriptor.runtime, 'modulestore'):
+                course_id = self.descriptor.runtime.course_id
+                if course_id:
+                    course = self.descriptor.runtime.modulestore.get_course(course_id)
+            if course:
+                licensable = course.licensable
+                if not video_license:
+                    video_license = course.license
```

Comment:
@esbeka can you please explain the logic of L246-249 here? I don't understand what you're saying... what I interpret as, "if not self.license, then the video license is the course.license". But what is "self.license"? That's on the video descriptor? What...??